### PR TITLE
Cust 1624 fixies to styling

### PR DESCRIPTION
**Pull request by darushka**
Updates in styling to match Design Spec.

### CODE REVIEW

File: client/app/material/md-button.scss [hunk 1]
```diff
@@ -1,28 +1,20 @@
...
-    font-weight: 600;
-  }
+    &.md-raised,
+    &.md-fab {
+
+      &:not([disabled]) {
+        background-color: $button-background-primary;
+        color: $inverse-font-color;
 
-  md-progress-circular {
```
Comment (jsmylnycky) [inline client/app/material/md-button.scss]: My concern here is that we're removing the markup to style the progress spinner thats used on the login page submit button.... we need to come up with a uniform way of applying that spinner to other forms, but thats what this was used for.

**Comment by jsmylnycky** [comment]
:+1:  Please merge squash, but know that we might need to get some work done quickly for the progress spinner before the launch Friday if needed :)

**Comment by darushka** [comment]
Merged squash https://github.com/peachworks/peach.css/commit/dd2715182b67cb82ddec9b602b644c43d0d508bc

**Comment by jsmylnycky** [comment]
Released v0.1.8

**Pull request closed without merging**